internal/app/exporter/exporter.go

```diff
@@ -335,10 +566,98 @@ func (c *rdsCollector) Describe(ch chan<- *prometheus.Desc) {
 	ch <- c.writeThroughput
 }
 
+func initGlobalVirables() {
+	if len(globalInstanceMetrics) > 0 {
+		globalInstanceMetrics = nil
+	}
+	if len(ServiceQuotaMetricsCounterMap) > 0 {
+		ServiceQuotaMetricsCounterMap = make(map[AccountRegionKey]ServiceQuotaMetricsCounter, 0)
+	}
+	if len(CloudwatchInstancesCounterMap) > 0 {
+		CloudwatchInstancesCounterMap = make(map[AccountRegionKey]CloudWatchMetricsCounter, 0)
+	}
+	if len(CloudWatchUsageCounterMap) > 0 {
+		CloudWatchUsageCounterMap = make(map[AccountRegionKey]CloudWatchUsageCounter, 0)
+	}
+	if len(EC2MetricsCounterMap) > 0 {
+		EC2MetricsCounterMap = make(map[AccountRegionKey]EC2MetricsCounter, 0)
+	}
+}
+
 // getMetrics collects and return all RDS metrics
 func (c *rdsCollector) fetchMetrics() error {
 	c.logger.Debug("received query")
 
+	if len(c.accountRegionClients) > 0 {
```

Comment:
The business logic is duplicated, while we may build a single loop with all AWS accounts to proceed.

In this implementation, AWS accounts will be retrieved one by one, whereas we could rely on Goroutine to collect them in parallel execution.